test/unit/k8s/deployment.test.ts

```diff
@@ -98,6 +100,40 @@ suite('K8s/deployment', () => {
         sandbox.restore();
     });
 
+    suite('DeploymentConfigNodeContributor', () => {
+        const parent = <ClusterExplorerV1.ClusterExplorerNode>{
+            metadata: undefined,
+            name: "comp1-app",
+            namespace: null,
+            nodeType: "resource",
+            resourceKind: {
+                abbreviation: "dc",
+                description: "",
+                displayName: "DeploymentConfigs",
+                label: "DeploymentConfigs",
+                manifestKind: "DeploymentConfig",
+                pluralDisplayName: "DeploymentConfigs"
+            }
+        };
+
+        setup(() => {
+            const api: k8s.API<k8s.KubectlV1> = {
```

Comment:
@sudhirverma I was asking to do something like this. 
You stub external api, create object instance and test actual methods, not prototype.

Reply:
Thanks, Denis :). I will check why test are failing.